docs/reference/modules/remote-clusters.asciidoc

```diff
@@ -5,104 +5,92 @@ clusters_. Remote clusters can be located in different datacenters or
 with {ccr} or searched by a local cluster using {ccs}.
 
-With <<xpack-ccr,{ccr}>>, you ingest data to an index on a remote cluster. This 
-_leader_ index is replicated to one or more read-only _follower_ indices on your local cluster. Creating a multi-cluster architecture with {ccr} enables you to 
-configure disaster recovery, bring data closer to your users, or establish a 
+[[remote-clusters-ccr]]
+[discrete]
+=== {ccr-cap}
+
+With <<xpack-ccr,{ccr}>>, you ingest data to an index on a remote cluster. This
+_leader_ index is replicated to one or more read-only _follower_ indices on your
+local cluster. Creating a multi-cluster architecture with {ccr} enables you to
+configure disaster recovery, bring data closer to your users, or establish a
 centralized reporting cluster to process reports locally.
 
+[[remote-clusters-ccs]]
+[discrete]
+=== {ccs-cap}
+
 <<modules-cross-cluster-search,{ccs-cap}>> enables you to run a search request
 against one or more remote clusters. This capability provides each region with a
 global view of all clusters, allowing you to send a search request from a local
 cluster and return results from all connected remote clusters. For full {ccs}
 capabilities, the local and remote cluster must be on the same
 {subscriptions}[subscription level].
 
-Enabling and configuring security is important on both local and remote
-clusters. When connecting a local cluster to remote clusters, an {es} superuser
-(such as the `elastic` user) on the local cluster gains total read access to the
-remote clusters. To use {ccr} and {ccs} safely, 
-<<remote-clusters-security,enable security>> on all connected clusters 
-and configure Transport Layer Security (TLS) on at least the transport level on
-every node.
-
-Furthermore, a local administrator at the operating system level
-with sufficient access to {es} configuration files and private keys can
-potentially take over a remote cluster. Ensure that your security strategy
-includes securing local _and_ remote clusters at the operating system level.
+[[add-remote-clusters]]
+[discrete]
+=== Add remote clusters
 
-To register a remote cluster,
-<<remote-clusters-connect,connect the local cluster>> to nodes in the
-remote cluster using sniff mode (default) or proxy mode. After registering
-remote clusters, <<remote-clusters-privileges,configure privileges>> for {ccr} 
-and {ccs}.
+To add remote clusters, you can choose between
+<<remote-clusters-security-models,two security models>> and
+<<sniff-proxy-modes,two connection modes>>. Both security models are compatible
+with either of the connection modes.
 
-[[sniff-mode]]
+[[remote-clusters-security-models]]
+[discrete]
+==== Security models
+
+API key based security model::
+beta:[]
+Uses an API key to authenticate and authorize cross-cluster operations to a 
+remote cluster. This model offers administrators of both the local and the 
+remote cluster fine-grained access controls. <<remote-clusters-api-key>>.
```

Comment:
I think we can also mention the 8.10 version requirement here to help people make quick decisions. 

Another thing is license. For now the new security model requires enterprise license. Do we need to mention it somewhere in the doc? Also ping @bytebilly for this question.

Reply:
We avoid mentioning specific license levels in the documentation, and refer folks to the subscriptions page instead. How about we add the following to the API key prerequisites (similar to what we have in [the audit logging docs](https://www.elastic.co/guide/en/elasticsearch/reference/master/enable-audit-logging.html)):

_Local and remote clusters must have an appropriate license. For more information, refer to https://www.elastic.co/subscriptions._

Reply:
Sounds great! Thanks for explaining.